### ISSUE TITLE
Add caption to map tab

### ISSUE THREAD

**Issue by LucyGwilliamAdmin** [open-sdg/open-sdg#900]
**Is your feature request related to a problem? Please describe.**
Both charts and tables have a caption - maps don't. See comparison of all three in images in #899 

**Describe the solution that you need**
Add caption above map in map tab which describes what the map shows

**Why do you need this feature?**
So that if someone takes a screenshot of map and uses it out of context, it's clear what it is showing, and also for consistency with the chart and table tabs

**Describe alternatives you've considered**
None

**Comment by LucyGwilliamAdmin** [comment]
This has also been raised by DAC

**URL:** http://sdgdev-813006012.eu-west-1.elb.amazonaws.com/dac2020/3-a-1/

There is no heading for the map to describe what information it is supposed to be displaying. It is assumed it is a geographical visualisation of the full data set, but this is confusing as the other two tabs have had categories selections specified such as gender breakdowns applied.

**Solution:**
Please implement a heading to describe what the map shows

**Comment by LucyGwilliamAdmin** [comment]
@brockfanning this was an existing ticket but I've also added info from DAC here

They've actually made a few points within just the one issue.

But I think the main things they are saying is make sure that the map has a title which says what the map shows and gives context to figures e.g. providing geographical area and unit type?

But they've also said in the description about text on the map not meeting colour contrast - do you want me to create a new issue for that?

**Comment by brockfanning** [comment]
@LucyGwilliamAdmin I think it depends on which text is failing contrast. I just added #946 which addresses the color contrast in the "attribution" section.
